src/css/variables.module.css

```diff
@@ -1,3 +1,68 @@
+@media large-tablet-breakpoint-query {
```

Comment:
The reason I didn't put a lib on the commit or branch name for this PR was because it doesn't correspond to any. It's just a general refactor.

Also, the variables added here correspond to variables used in previous PRs. The reason I'm putting them in all at once here is because separating them by PR would have taken extra time unnecessarily

Reply:
Cool